lib/ansible/modules/system/known_hosts.py

```diff
@@ -145,7 +148,7 @@ def enforce_state(module, params):
                 module.fail_json(msg="Failed to read %s: %s" % \
                                      (path,str(e)))
         try:
-            outf=tempfile.NamedTemporaryFile(dir=os.path.dirname(path))
+            outf = tempfile.NamedTemporaryFile(mode='w+', dir=os.path.dirname(path))
```

Comment:
I keep writing extensive commit messages and then forgetting reviewers have to take extra steps to see them in GitHub's review UI.

(And when I'm reviewing PRs for projects I maintain, I tend to ignore commit messages and just look at the final diff.)